specification/cognitiveservices/OpenAI.Inference/models/completions.create.cadl

```diff
@@ -163,3 +165,14 @@ model CompletionsLogProbsModel {
     @doc("Text offset")
     text_offset?: int32[];
 }
+
+@Versioning.added(Azure.OpenAI.ServiceApiVersions.v2022_12_01)
```

Comment:
There were not added in this version, as they were already in the previous one. I don't think this PR should use `added` at all actually

Reply:
It appears we were missing it for `Completions`